aten/src/ATen/native/TensorFactories.cpp

```diff
@@ -274,6 +274,17 @@ Tensor logspace(
 
 // ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ ones ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+#ifdef NAMEDTENSOR_ENABLED
+Tensor ones(
+    IntArrayRef size,
+    at::optional<DimnameList> names,
```

Comment:
I know we don't plan to support an ordered-dict option because of python2, but is there any chance we'll want to support an option to construct named tensors in C++ using an ordered map? or would the syntax be just as unwieldy as python2?

Reply:
There are no plans for named tensor support in C++ in the near future.